README.md

```diff
@@ -40,15 +40,35 @@ prioritize these additional components
 
 There are two components to this process:
 
-1. [Instrumenting the Wasm code](#instrumenting-wasm-modules)
-2. [Including a runtime/host SDK](#including-a-runtime-sdk)
+1. [Including a runtime/host SDK](#including-a-runtime-sdk)
+2. [Instrumenting the Wasm code](#instrumenting-wasm-modules)
+
+## Including a runtime SDK and an Adapter
+
+First you should choose a Host SDK corresponding to your host application's language and Wasm runtime.
+The Host SDK captures raw observability events from the running Wasm module and sends them to an adapter.
+You must choose an adapter based on where you want your data to go. At the moment, we support a few systems out of the box.
+In the future we will support a lot more and will have more community driven options. If you don't see support for your
+favorite observability tools feel free to reach out to us at ([support@dylibso.com](mailto:support@dylibso.com)).
+
+Each language includes some examples demonstrating use with different adapters.
+You can view these examples here:
+
+- [Rust](rust/examples)
+- [Go](go/bin)
+- [Js](js/packages)
```

Comment:
Is this the right spot?

Reply:
we need to do a full clean-up pass over the repo and readme.. i wouldnt let it block you from merging this if you're ready